docs/general/session.md

```diff
@@ -35,4 +35,41 @@ backends can link the two sessions.
 <!-- END AUTOGENERATED TEXT -->
 <!-- endsemconv -->
 
+## Session Events
+
+### Session Start Event
+
+![Experimental](https://img.shields.io/badge/-experimental-blue)
+
+`event.name`=`session.start`
+
+For instrumentation that tracks user behavior during user sessions, a `session.start` event MUST be emitted
+every time a session is created. When a new session is created as a continuation of a prior session,
+the `session.previous_id` SHOULD be included in the event. The values of `session.id` and `session.previous_id`
+MUST be different.
+
+When the `session.start` event contains both `session.id` and `session.previous_id` fields, the event then implies
+that the previous session has ended. If the session ID in `session.previous_id` has not yet ended via explicit
+`session.end` event, then the receiver SHOULD treat this continuation event as semantically equivalent to
+`session.end(session.previous_id)` and `session.start(session.id)`.
+
+| Body field                                                    | Type   | Description                                          | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability                                                        |
+|---------------------------------------------------------------|--------|------------------------------------------------------|-------------------------------------------------------------------------------------------------------|------------------------------------------------------------------|
+| [`session.id`](/docs/attributes-registry/session.md)          | string | The ID of the new session being started.             | `Required`                                                                                            | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`session.previous_id`](/docs/attributes-registry/session.md) | string | The previous `session.id` for this user, when known. | `Conditionally Required`                                                                              | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+### Session End Event
+
+![Experimental](https://img.shields.io/badge/-experimental-blue)
+
+`event.name`=`session.end`
+
+For instrumentation that tracks user behavior during user sessions, a `session.end` event SHOULD be emitted
```

Comment:
nit: is it really an instrumentation that will emit these events? If I am not wrong, it would be some SDK level software. If correct, please consider using another term for instrumentation.

Reply:
The word `instrumentation` is used there in regard to software that is tracking user behavior, not in regard to the thing that generates events.